From 2559f096c73942d9c9c2d96ed5170b01d634f412 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Tue, 16 Oct 2018 10:31:18 +0800
Subject: [PATCH] clk: rockchip: rk1808: mark hsclk_imem as critical clock

hsclk_imem is pd_npu and pd_imem share niu clk.

Change-Id: I56e06edeb16340b2df7b1033c4fd65a61e22054e
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 drivers/clk/rockchip/clk-rk1808.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/rockchip/clk-rk1808.c b/drivers/clk/rockchip/clk-rk1808.c
index fc2b58330b40..8cf5f5a9aa3c 100644
--- a/drivers/clk/rockchip/clk-rk1808.c
+++ b/drivers/clk/rockchip/clk-rk1808.c
@@ -1156,6 +1156,7 @@ static const char *const rk1808_critical_clocks[] __initconst = {
 	"pclk_top_pre",
 	"pclk_ddr_grf",
 	"aclk_gic",
+	"hsclk_imem",
 };
 
 static void __iomem *rk1808_cru_base;
-- 
2.35.3

